src/thematic/0_generic/sfp-001_vervanging-terrest_nl/10_samenvatting.Rmd

```diff
@@ -0,0 +1,14 @@
+# Samenvatting
+
+<!--
+Een samenvattende opsomming van de te volgen stappen. Gebruik eventueel een stroomschema indien dit het protocol kan verduidelijken.
+-->
+
+Samenvattend bestaat dit veldprotocol uit volgende stappen:
+
+1. als voorbereiding van het veldwerk laad je de nodige gegevens in een veldcomputer zodat je op terrein de mogelijke vervangpuntenen de volgorde ervan kan visualiseren
+2. op terrein start je de vervangingsprocedure als een steekproefeenheid niet geschikt blijkt
```

Comment:
> op terrein start je de vervangingsprocedure als een steekproefeenheid niet geschikt blijkt

Dit is geen stap van het vervangprotocol, maar het toepassingsgebied ervan. Het starten van de vervangprocedure lijkt me _wel_ een stap in het unit-selection protocol, dat dus doorkoppelt naar het vervangprotocol.

Reply:
Eens herhalen kan geen kwaad, denk ik.